_layouts/article.html

```diff
@@ -21,7 +21,7 @@
           </div>
           <div class="overlay-card-content">
             <p class="overlay-card-category">{{ page.category.title }}</p>
-            <h2 class="overlay-card-title overlay-card-heading">{{ page.title | truncate: 70 }}</h2>
+            <h2 data-media-title="{{page.title}}" data-media-id="{{page.id}}" class="overlay-card-title overlay-card-heading">{{ page.title | truncate: 70 }}</h2>
```

Comment:
This may not be relevant but the Contentful ID for each page is already exposed in a meta_tag...  

<img width="782" alt="Screen Shot 2022-09-26 at 9 13 24 AM" src="https://user-images.githubusercontent.com/50378/192285825-c21aa646-1254-434b-b10a-374037ab2311.png">